manage-data/lifecycle/index-lifecycle-management/rollover.md

```diff
@@ -9,50 +9,91 @@ products:
 
 # Rollover [index-rollover]
 
-When indexing time series data like logs or metrics, you can’t write to a single index indefinitely. To meet your indexing and search performance requirements and manage resource usage, you write to an index until some threshold is met and then create a new index and start writing to it instead. Using rolling indices enables you to:
+In {{es}}, the [rollover action](elasticsearch://reference/elasticsearch/index-lifecycle-actions/ilm-rollover.md) replaces your active write index with a new one whenever your index grows too large, too old, or stores too many documents.
+This is particularly useful for time-series data, such as logs or metrics, where index growth is continuous and performance can degrade over time.
 
-* Optimize the active index for high ingest rates on high-performance *hot* nodes.
-* Optimize for search performance on *warm* nodes.
-* Shift older, less frequently accessed data to less expensive *cold* nodes,
-* Delete data according to your retention policies by removing entire indices.
+Without rollover, a single index can grow until searches slow down, shard relocation takes hours, and nodes run low on disk space.
 
-We recommend using [data streams](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-create-data-stream) to manage time series data. Data streams automatically track the write index while keeping configuration to a minimum.
+The rollover feature is an important part of how [index lifecycle](../index-lifecycle-management/index-lifecycle.md) and [data stream lifecycles](../data-stream.md) work to keep your indices fast and manageable. By switching the write target of an index, the rollover action provides the following benefits:
 
-Each data stream requires an [index template](../../data-store/templates.md) that contains:
+* **Automation** - works with ILM to remove manual index rotation tasks and allows for granular control over retention cycles
+* **Optimized performance** - keeps shard sizes within recommended limits (10-50 GB)
+* **Queries run faster** - improves search performance
 
-* A name or wildcard (`*`) pattern for the data stream.
-* The data stream’s timestamp field. This field must be mapped as a [`date`](elasticsearch://reference/elasticsearch/mapping-reference/date.md) or [`date_nanos`](elasticsearch://reference/elasticsearch/mapping-reference/date_nanos.md) field data type and must be included in every document indexed to the data stream.
-* The mappings and settings applied to each backing index when it’s created.
+Rollover can be triggered via the [API](https://www.elastic.co/docs/api/doc/elasticsearch/operation/operation-indices-rollover) or ILM.
```

Comment:
Again we only mention ILM, though DLM also triggers rollover. We should mention it here.